tests/onnxruntime/test_modeling.py

```diff
@@ -1946,6 +1946,7 @@ class ORTModelForCausalLMIntegrationTest(ORTModelTestMixin):
         "bloom",
         "codegen",
         "gpt2",
+        "gpt_bigcode",
```

Comment:
Have you run the tests on GPU?

Reply:
Just triggered the CI on GPU. Only tested on GPU for bigcode on my local.